app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/presenter/SubscriberDetailPresenterImpl.java

```diff
@@ -51,14 +51,18 @@ public SubscriberDetailPresenterImpl(SubscriberDetailView view, User userData, L
 
     public void onFinishedUser(User userItem) {
         userData = userItem;
-        view.showSubscriberDetails(userData);
+        if (view != null) {
+            view.showSubscriberDetails(userData);
+            view.toggleProgressIndicator(false);
```

Comment:
The natural order of the experience is: hide the loading then show the data, so, put `view.toggleProgressIndicator(false);` first then show the data.